README.md

```diff
@@ -662,6 +662,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 ## Icon
 
 - [nvim-tree/nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons) - A Lua fork of [vim-devicons](https://github.com/ryanoasis/vim-devicons).
+- [echasnovski/mini.nvim#mini.icons](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-icons.md) - Module of `mini.nvim` meant as a general icon provider. Uses fixed set of highlight groups. Supports various categories, icon and style customizations, caching for performance. Integrates with Neovim's filetype matching.
```

Comment:
```suggestion
- [echasnovski/mini.nvim#mini.icons](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-icons.md) - Module of `mini.nvim` meant as a general icon provider. Uses fixed set of highlight groups. Supports various categories, icon and style customizations, caching for performance. Integrates with filetype matching.
```

Reply:
Is "built-in filetype matching" OK?

Reply:
Sure! I just would like to avoid all the mentions of Neovim since it's obvious that everything relates to it.